File: Frontend/FrontMedicalApplication/src/app/core/dtos/billing.dto.ts
export class BillingDto {
    billingId?: number;
    patientId: number;
    amount: number;
    dateIssued: Date;
    paymentStatus: string;
  
    constructor(
      patientId: number,
      amount: number,
      dateIssued: Date = new Date(),
      paymentStatus: string = 'Pending',
      billingId?: number
    ) {
      this.patientId = patientId;
      this.amount = amount;
      this.dateIssued = dateIssued;
      this.paymentStatus = paymentStatus;
      if (billingId) {
        this.billingId = billingId;
      }
    }
  }
  

File: Backend/WebMedicalApplication/BackendMedicalApplication/DTo/BillingDto.cs
namespace BackendMedicalApplication.DTo
{
    public class BillingDto
    {
        public int PatientId { get; set; }
        public decimal Amount { get; set; }
        public DateTime DateIssued { get; set; }
        public string PaymentStatus { get; set; }
    }

}
